components/Header/index.js

```diff
@@ -83,6 +128,31 @@ const Header = ({ logOut }) => {
               <Text style={styles.calculationText}>{`${I18n.t('header.volunteerSince')}\n${volunteerDate}`}</Text>
               <Text style={styles.calculationText}>{`${I18n.t('header.surveysCollected')}\n${surveyCount}`}</Text>
             </View>
+            {offlineForms ? (
+              <Button onPress={postOffline}>Submit Offline Forms</Button>
+            ) : (
+              <Button disabled>Submit Offline Forms</Button>
+            )}
+            {failedSubmission && (
+              <View>
+                <Text style={styles.calculationText}>Failed Attempt Submitting Offline Forms.</Text>
+                <Text style={{ alignSelf: 'center' }}>Please ensure that you are connected to the internet and try again.</Text>
+                <Button onPress={() => setFailedSubmission(false)}>Ok</Button>
+              </View>
+            )}
+            {successfullSubmission && (
+              <View>
+                <Text style={styles.calculationText}>Success!</Text>
+                <Text style={{ alignSelf: 'center' }}>
+                  You have just submitted
+                  {' '}
+                  {offlineFormCount}
+                  {' '}
+                  forms!
+                </Text>
+                <Button onPress={() => setSuccessfullSubmission(false)}>Ok</Button>
```

Comment:
and all this translated as well

and instead of managing two states `successfulSubmission` and `failedSubmission`, just use one  (maybe just call it `submissionSuccess` or w/e


then just use `submissionSuccess == false  && (` in the render instead of failedSubmission